app/graphql/types/room_type.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+module Types
+  class RoomType < Types::BaseObject
+    field :id, ID, null: false
+    field :name, String, null: false
```

Comment:
It looks like Host might be missing here - but I think my PR might fix it. I'll merge yours and see what happens.